the-collab-lab/tcl-17-smart-shopping-list
# Bl delete token words

## Trigger Warning 
You do not need to view the removed words to approve the PR request. If you decide to navigate to the `files changed` view, you will see the triggering or offensive words that have been removed from the project. If you do not want to see these words, please avoid navigating to the `files changed` view.

## Description
This PR will remove triggering and offensive words from the `token.js` words array. The removal of these words is in alignment with TCL's mission of inclusion and the creation of a safe space for all.

## Related Issue
n/a

## Acceptance Criteria
n/a


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|   ✓ | Other              |

## Updates

### Before
No changes to the UI



### After
No changes to the UI



## Testing Steps / QA Criteria
**Please only follow these steps if you are comfortable being exposed to triggering or offensive words.** To confirm that these words have been removed from the file you can 
1. pull the changes down locally using `git pull BL-delete-token-words` 
2. navigate to that branch with `git checkout BL-delete-token-words` 
3. use `ctrl+F` (on Windows) or `cmd+F` (on Mac) to pull up the search feature and enter any of the triggering words (which can be found in the `files changed` view in github.
